# PSC_CUNY55_Rochman
**Characterizing the Urban Rat Gut Virome: Preliminary Results**

Rat feces were collected from 5 sites across the Bronx, Queens, and Manhattan (Fig. A) and frozen for bulk preparation. Samples were thawed overnight, weighed, and homogenized. Lysis buffer was added to 50mg of each homogenized sample and whole RNA extraction was performed. Total RNA was quantified and a pooled sample was obtained extracting an equal amount from each sample. Site-normalized RNA extraction was performed in favor of pellet-normalization or bulk extraction per the expectation that this would maximize the viral diversity. Whole RNA was shipped to a commercial sequencing provider who performed total RNA library preparation with both bacterial and ribosomal RNA depletion. Raw reads were analyzed in house.

Reads were assembled into contigs using rnaSPAdes. All contigs with coverage of 10 or greater (~40k) were translated into all six frames and queried via pairwise similarity search against all ICTV Orthoriboviria species exemplars and profile search against RdRP profiles. Contigs with a hit to either database were then queried via pairwise similarity against NCBI mammalian reference genomes and the Prok2311 prokaryote reference genome database. Contigs with hits to ICTV, without significant overlap in alignments to mammalian or prokaryote sequences or superior e-value in the case of overlap, and a minimum e-value of 10e-6 were assigned metadata corresponding to the best-hit viral protein. Remaining contigs with hits to an RdRP profile with no, more specific, ICTV assignment were queried against nr via psiblast. Each contig (17) met the e-value threshold and was assigned the metadata corresponding to the best hit. This procedure resulted in 446 proteins of putative RNA viral origin from at least 148 unique strains. Raw reads were then aligned to the corresponding contigs using SMALT and the number of perfect matches (Phred of 60) were tabulated. For each strain, the contig with the greatest number of mapped reads was selected as the representative and normalized by the length of the sequence. The host for each strain was estimated through a semi-automated procedure identifying keywords in the GenBank file. Putative hosts were classified into six groups: insect, microbe, diet-plant, diet-animal, fungi, and uncharacterized as well as direct-infection. Strains classified as diet-animal and direct-infection were mapped to nonmammalian vertebrates and mammals respectively. The majority of direct-infection hits were human or rodent specific.

![Alt Text](psc55figure.png)
 
**A:** Map of collection sites. **B:** Relative abundance of viral taxa with host/source estimation for 30 most abundant strains and all strains estimated to be direct infections of the rats surveilled.

As expected, nearly half of the viral copies were predicted to be from diet or microbial hosts. Among these, it is notable that two of the putative microbial hosts are eukaryotes, algae, including the diatom Chaetoceros tenuissimus. Initially unexpected by the study team, nearly one third of the viral copies are likely contributed by insect hosts which could both be present at the collection site as scavengers and consumed by the rat. The identified host for the most highly abundant virus, globally, is the red fire ant. This is a striking finding either reflecting spillover infection to native ant species or the identification of invasive red fire ants in NYC, subject to a rapidly expanding geographic range extending to VA, with reports of recent sightings further North. Two closely related viruses implicated in honeybee colony loss were also identified in high abundance. Abundant plant fungal viruses were also identified, reflecting the late stage of decomposition for the plants consumed by the rat.

Of the 25 viruses identified that likely directly infected a surveilled rat, cardioviruses represent nearly half of the viral copies. Cardiovirus is a well-established genus including groups of both human and rodent pathogens. Of epidemiological relevance but also expected, at least one rodent coronavirus was identified with the closest relative being an alphacoronavirus first characterized in China in 2015. Additionally, norovirus was isolated. The majority of the pathogens identified likely infrequently result in cross-species transmission; however, it is notable that, in some cases, the closest reference genomes were human isolates. Of potential concern, is the putative detection of Rubella. The only identified natural reservoir for Rubella is the human host; however, diverse mammalian relatives of the virus have been identified. We are conducting more detailed phylogenetic reconstructions for all available genomes to identify the placement of this isolate.
